server/odc-service/src/main/java/com/oceanbase/odc/service/datatransfer/DataTransferService.java

```diff
@@ -284,27 +270,12 @@ public Map<ObjectType, Set<String>> getExportObjectNames(
 
         try (DBObjectNameAccessor accessor = DBObjectNameAccessor.getInstance(connection, database.getName())) {
             Map<ObjectType, Set<String>> returnVal = new HashMap<>();
-            if (CollectionUtils.isNotEmpty(objectTypes)) {
-                for (ObjectType objectType : objectTypes) {
-                    returnVal.putIfAbsent(objectType, accessor.getObjectNames(objectType));
-                }
-                return returnVal;
+            if (CollectionUtils.isEmpty(objectTypes)) {
```

Comment:
firstly, we get the supported type from datatransferExtension.
secondly, if the objecttypes is not empty then take the intersection of objectTypes and the returning of datatransferextension.
thirdly, use SchemaExtension to get the name list of them

Reply:
done. Use SchemaPlugin instead of SchemaAccessors in `DBSchameNameAccessor` .